src/compiler/transformers/test/transpile.ts

```diff
@@ -108,10 +108,23 @@ export function transpileModule(
     styleImportData: 'queryparams',
   };
 
+  const collection: d.CollectionCompilerMeta = {
```

Comment:
I was running into an issue I think because of these lines:

https://github.com/ionic-team/stencil/blob/d0292e0c37d31b87a72a2b83434c8dab90aff120/src/compiler/transformers/static-to-meta/visitor.ts#L41-L48

basically if the `collection` argument isn't present then some values arent' set, and when I was debugging tests I was trying to figure something out about the collection - we don't need to actually keep this change around though